packages/pwa-kit-dev/src/utils/script-utils.ts

```diff
@@ -279,6 +279,65 @@ export class CloudAPIClient {
         const data = await res.json()
         return data['token']
     }
+
+    /** Polls MRT for deployment status every 60 seconds. */
+    async waitForDeploy(project: string, environment: string): Promise<void> {
+        return new Promise((resolve, reject) => {
+            const delay = 60e3 // 60 seconds
```

Comment:
60 seconds is probably ok but it might also be a bit long (would 45 seconds be better, not sure)? Deploys that don't involve rebuilding the CDN don't take up a lot of time (probably around 2 min?)

Reply:
I've decreased the delay to 30 seconds.

Reply:
This resolves a promise when the server indicated "DONE", right? So why are we worried about exiting early if the only chance of taking "too long" is that there is a real problem in the server layer?

Reply:
It's not so much as worried about exiting early but more not having to wait as long for the polling to say the build is complete once MRT is done. This shaves off 30 seconds of wait time for those who run the command locally and are not expecting a long build (as is the case when MRT skips rebuilding the CDN)